{"name": "test_login_0_user_wrongFormat[data8]", "status": "passed", "description": "\n        异常用例 - 登录失败-数据不正确\n        步骤:\n         1.输入用户名：XX 密码：XX 点击登录\n         2. 断言： 登录页面 提示 - XXX\n              登录页面中 - 获取提示框的文本内容\n              比对文本内容 与 期望的值 是否相等。\n        :param data:\n        :param access_web1:\n        :return:\n        ", "attachments": [{"name": "log", "source": "3558f4cb-3dbe-40c3-ad51-9068f6bb53f8-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'username': 55173206, 'passwd': 55173206, 'check': '请29分钟后再试或使用忘记密码找回密码', 'explain': '账号密码错误'}"}], "start": 1596259452212, "stop": 1596259453025, "uuid": "e5fa79a2-bb92-4cbc-a53f-e759034a1cde", "historyId": "8fce9bf0147040ef7b95572d0ea5c2e1", "testCaseId": "fe0639cb41592b40517b5dd1e2687260", "fullName": "test_login_xcfb.TestLoginXcfb#test_login_0_user_wrongFormat", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('refresh_page1')"}, {"name": "suite", "value": "test_login_xcfb"}, {"name": "subSuite", "value": "TestLoginXcfb"}, {"name": "host", "value": "DESKTOP-6E8RQN3"}, {"name": "thread", "value": "33204-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login_xcfb"}]}